lib/asset.js

```diff
@@ -2,14 +2,24 @@
 
 const _ = require('lodash');
 const crypto = require('crypto');
+const validator = require('validator');
+
+const config = require('./config');
+
+function _coerce (s) {
```

Comment:
Everything that is checked with `validator.js` has to be coerced to string (see their docs). The coercion can't be inside `_sanitizeText` because not every validated metadata will be sanitized as text.

That being said, I'm not really liking `validator.js` for all validations. I think we can use `lodash` and custom functions for many of them.